### ISSUE TITLE
Kee autofills all fields

### ISSUE THREAD

**Issue by levicki** [kee-org/browser-addon#239]
I am using Kee in Chrome.

When I visit a website for which I have a saved login in my KeePass database:
https://www.euprava.gov.rs/registracija

Kee mistakenly pre-populates the site search field with my username (I removed the username from the screenshot, but you can see Kee's icon on the field):
![image](https://user-images.githubusercontent.com/16415478/60924242-f9beaa00-a2a0-11e9-9e23-333bb9de2359.png)

I am using default URL matching configuration and matching password entry in KeePass has this URL:
https://www.euprava.gov.rs/prijava

1. Kee probably shouldn't even try to autofill on different URL path, unless it is matching on domain only by default?
2. The search field in question is not even part of the form element.
3. I could filter by name (which is `ctl02$ctl06$ctl05$txtSearchSimple`) or by ID (which is `ctl02_ctl06_ctl05_txtSearchSimple`), but shouldn't it already match that both values contain the word "search"?

TL;DR -- I would like to avoid having to maintain a list of custom field names if they already contain a word "Search" in them.

**Comment by levicki** [comment]
@luckyrat 

I changed the subject because for me Kee is auto-filling any field it sees with password for that site and as such it is totally unusable in Chrome.

**Comment by luckyrat** [comment]
If this is still relevant in Kee 3.6, feel free to search and discuss on the forum so we can determine how best to help: https://forum.kee.pm
